src/app/space/create/apps/components/resource-card.component.ts

```diff
@@ -25,14 +28,24 @@ export class ResourceCardComponent implements OnInit {
   usedPercent: number;
   unusedPercent: number;
 
+  private readonly unsubscribe: Subject<void> = new Subject<void>();
```

Comment:
Why are you creating a variable called `unsubscribe`? RxJS has a function on `Observable` called `unsubscribe()`.

What are you using `Subject` for? This does not look the correct use of this.

Reply:
It's taken from the example here:

http://brianflove.com/2016/12/11/anguar-2-unsubscribe-observables/

It looked like a nice solution for the possibility of having to manage unsubscribing multiple Observables at a time, although it's definitely overkill here where we only have one.

Reply:
You are right, unsubscribe is a part of subscription. 

I would rather you figure out the right way to use Observables then use an example that is not doing what you really need. You are not in a rush.

Reply:
The ways I've discovered to manage Observables are:

1) use the `async` pipe in the template, which documentation says will have Angular properly unsubscribe when the component is destroyed along with avoiding having to write TS to assign values to properties etc.

2) keep reference to each `Subscription` after subscribing, and call `unsubscribe` on each on destroy. Possibly by keeping the subscriptions in a collection and `forEach`ing it.

3) use an operator like `first` so that the Subscription is automatically closed as it's "finite"

4) use an operator like `takeWhile` or `takeUntil` to use some other event stream (ex. component destruction) to signal when the primary event stream is no longer needed (the original approach in this PR)

Are there any others that I've missed? I've listed them in the order that I believe they're preferred, judging by other examples in the codebase and this conversation.

Reply:
I can't think of any but I'm not an RxJS expert.